ibexa/twig-components/4.6/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "aliases": [],
+    "bundles": {
+        "Ibexa\\Bundle\\TwigComponents\\IbexaTwigComponentsBundle": ["all"]
```

Comment:
Does the bundle order matter in some cases? If yes, this should be placed inside https://github.com/ibexa/recipes-dev/blob/master/ibexa/commerce/4.6/manifest.json

Reply:
After consulting it with Adam, seems that it should be fine